scripts/clean_data.py

```diff
@@ -0,0 +1,29 @@
+import click
+import pandas as pd
+
+@click.command()
+@click.option('--input_path', type=str, help="Path to input CSV file to be cleaned.")
+@click.option('--output_path', type=str, help="Path where the cleaned file will be saved.")
+def clean_data(input_path, output_path):
```

Comment:
I believe the standard practice is to keep this as `main()`, but I once the correct function is called below, and the script command works, I do not think it is a requirement.

Let me know what you think.